### DISCUSSION TITLE
Litmus | workflow not working | "Please check the configuration, the workflow was not able to start"

### DISCUSSION THREAD

**Discussion by antonyjohnson-js** [litmuschaos/litmus#3541] [Q&A] [answered]
Hi There,

I'm a newbie to litmus and I deployed the litmus in one of the k8s ROKS clusters, i try to use a predefined template for testing purposes once the experiment is created it shows this error "Please check the configuration, the workflow was not able to start" could you please help me to resolve this.
i have used "podtato-head" in this workflow, FYI none of the workflows from ChosHub is also not working for me. Can anyone help with this

and I found this error while creating new workflows, attached the same for your reference 

[error-log.txt](https://github.com/litmuschaos/litmus/files/8410185/error-log.txt)

**Comment by neelanjan00** [comment]
CC: @Jonsy13 can you PTAL?

**Comment by neelanjan00** [comment, marked as answer]
Closing this discussion, I hope you have been able to figure it out.
